scipy/stats/_morestats.py

```diff
@@ -4061,26 +4076,37 @@ def directionalmean(samples, *, axis=0, normalize=True):
 
     .. code-block:: python
 
-        mean=samples.mean(axis=0)
-        directionalmean = mean/np.linalg.norm(mean)
+        mean = samples.mean(axis=0)
+        mean_resultant_length = np.linalg.norm(mean)
+        mean_resultant = mean / mean_resultant_length
 
     This definition is appropriate for *directional* data (i.e. vector data
     for which the magnitude of each observation is irrelevant) but not
     for *axial* data (i.e. vector data for which the magnitude and *sign* of
     each observation is irrelevant).
 
+    The function also returns the length of mean resultant vector which
+    can be used to calculate directional variance. Several definitions of
+    directional variance have been proposed e.g. ``Var(z) = 1 - R`` (where
+    ``R`` is the length of the mean resultant vector),
+    ``Var(z) = 1 - R**2`` and ``Var(z) = 2 * (1 - R)``. `directional_stats`
+    instead returns the length of the mean resultant vector so one can
+    choose the appropriate definition based on the use case.
+
     References
     ----------
     .. [1] Mardia, Jupp. (2000). *Directional Statistics*
        (p. 163). Wiley.
```

Comment:
```suggestion
       (p. 163). Wiley.

```
This blank line could solve the doc rendering issue (has sometimes worked for me in special examples at least).

Reply:
Done.